File: trail0/좌우로 반복하며 출력/print-in-left-right-reverse.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for(int i = 0; i < n; i++){
            if(i % 2 == 0){
                int cnt = 1;
                for(int j = 0; j < n; j++){
                    System.out.print(cnt);
                    cnt++;
                }
                System.out.println();
            }else{
                int cnt = n;
                for(int j = n-1; j > -1; j--){
                    System.out.print(cnt);
                    cnt--;
                }
                System.out.println();
            }
        }
    }
}

File: trail0/좌우로 반복하며 출력/print-in-left-right-reverse.py
n = int(input())
for i in range(1, n+1):
    if i % 2 != 0:
        cnt = 0
        for j in range(n):
            cnt += 1
            print(cnt, end="")
        print()
    else:
        cnt = n
        for j in range(n):
            print(cnt, end="")
            cnt -= 1
        print()